src/components/browse/ProductCard.tsx

```diff
@@ -1,31 +1,42 @@
+import { MouseEvent } from "react";
+import { Link } from "react-router-dom";
 import { priceInRands } from "../../utils";
 
 const ProductCard = ({
+  id,
   image,
   title,
   price,
   handleAddToCart,
 }: {
+  id: number;
   image: string;
   title: string;
   price: number;
   handleAddToCart: () => void;
 }) => {
+  const handleAddtoCartHere = (e: MouseEvent<HTMLButtonElement>) => {
```

Comment:
Just a question, what default behaviour are you trying to prevent?

Reply:
The navigation to the product info tab when the add to cart button is clicked (otherwise clicking on add to cart adds a product to the cart and also navigates to the product info page - which shouldnt happen)